seacrowd/sea_datasets/tcope/tcope.py

```diff
@@ -0,0 +1,163 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """
+@article{gonzales_broadening_2023,
+  author    = {Gonzales, Wilkinson Daniel Wong},
+  title     = {Broadening horizons in the diachronic and sociolinguisstic study of
+  Philippine Englishes with the Twitter Corpus of Philippine Englishes (TCOPE)},
+  journal   = {English World-Wide},
+  year      = {2023},
+  url       = {https://osf.io/k3qzx},
+  doi       = {10.17605/OSF.IO/3Q5PW},
+}
+"""
+
+_LOCAL = False
+_LANGUAGES = ["eng", "fil"]
+_DATASETNAME = "tcope"
+_DESCRIPTION = """
+The TCOPE dataset consists of 1,048,576 public tweets (amounting to about 13.5 million words) collected from 13 major cities from the Philippines.
+Tweets are either purely in English or involve code-switching between English and Filipino.
+Tweets are tagged for part-of-speech and dependency parsing using spaCy. Tweets collected are from 2010 to 2021.
+The publicly available dataset is only a random sample (10%) from the whole TCOPE dataset, which consist of roughly 27 million tweets
+(amounting to about 135 million words) collected from 29 major cities during the same date range.
+"""
+
+_HOMEPAGE = "https://osf.io/3q5pw/wiki/home/"
+_LICENSE = Licenses.CC0_1_0.value
+_URL = "https://files.osf.io/v1/resources/3q5pw/providers/osfstorage/63737a5b0e715d3616a998f7"
+
+_SUPPORTED_TASKS = [Tasks.POS_TAGGING]
```

Comment:
I think we shd add `DEPENDENCY_PARSING` task regardless of no implementation in SEACrowd Schema.

For the testing purpose, I can test it separately by commenting the `DEPENDENCY_PARSING` for SEACrowd Schema in POS Tagging and test for `source` implementation for both tasks.

wdyt, @elyanah-aco?

cc @holylovenia @SamuelCahyawijaya